src/libs/NumberCovert.js

```diff
@@ -0,0 +1,90 @@
+import lodashGet from 'lodash/get';
+import loadashTrim from 'lodash/trim';
+import loadashTrimStart from 'lodash/trimStart';
+import loadashToString from 'lodash/toString';
+import loadashIncludes from 'lodash/includes';
+import loadashStartsWith from 'lodash/startsWith';
+import loadashPadStart from 'lodash/padStart';
+import Log from './Log';
+import Config from '../CONFIG';
+import translations from '../languages/translations';
+
+function showErrorAndReturn(locale) {
```

Comment:
I think we can remove this function, as I don't think logging an alert to the server is necessary here and I don't think we should be returning an error string from the functions below. cc'ing @iwiznia for review to confirm.

Reply:
I have removed this function in new commit.